nodeup/pkg/model/secrets.go

```diff
@@ -58,6 +58,26 @@ func (b *SecretBuilder) Build(c *fi.ModelBuilderContext) error {
 		c.AddTask(t)
 	}
 
+	if b.SecretStore != nil {
+		key := "nodedockercfg"
+		dockercfg, err := b.SecretStore.Secret(key)
+		if err != nil {
+			return err
+		}
+		if dockercfg == nil {
+			return fmt.Errorf("node docker config not found: %q", key)
```

Comment:
oops I meant to have this as a guard when nodedockercfg is stored but the lookup failed.  I think you're right, I'll just remove it and rely on the string check.